packages/protocol/contracts/L1/libs/LibProposing.sol

```diff
@@ -80,6 +82,24 @@ library LibProposing {
             revert L1_TOO_MANY_BLOCKS();
         }
 
+        bytes32 blobOrTxListHash;
```

Comment:
I prefer to use `blobHash` and you can add the same comment for it:  `//or txListHash (if Blob not yet supported)`